tests/e2e/system_test/test_run_in_bg.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from ocs_ci.helpers import helpers
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.benchmark_operator_fio import BenchmarkOperatorFIO
+from ocs_ci.ocs.resources.pod import cal_md5sum
+
+log = logging.getLogger(__name__)
+
+
+class TestRunInBg(E2ETest):
```

Comment:
should we replace existing approaches of running IO in other tests to use benchmark operator?